packages/documentation/src/pages/getting-started/common-tasks/unit-tests.mdx

```diff
@@ -0,0 +1,137 @@
+## Unit Tests
```

Comment:
This doc is more about how our unit tests work than how to write one. So I think it should go in the architecture section (there's a "How do unit tests work?" placeholder)